light/errors.go

```diff
@@ -5,9 +5,42 @@ import (
 	"fmt"
 	"time"
 
+	cmtbytes "github.com/cometbft/cometbft/libs/bytes"
+	cmtmath "github.com/cometbft/cometbft/libs/math"
+	"github.com/cometbft/cometbft/light/provider"
 	"github.com/cometbft/cometbft/types"
 )
 
+var (
+
+	// ErrFailedHeaderCrossReferencing is returned when the detector was not able to cross reference the header
+	// with any of the connected witnesses.
+	ErrFailedHeaderCrossReferencing = errors.New("all witnesses have either not responded, don't have the " +
+		" blocks or sent invalid blocks. You should look to change your witnesses" +
+		"  or review the light client's logs for more information")
```

Comment:
Cut back on spaces a little:

```suggestion
		"blocks or sent invalid blocks. You should look to change your witnesses" +
		"or review the light client's logs for more information")
```